tsl/src/compression/compression.c

```diff
@@ -60,6 +60,31 @@
 #include "ts_catalog/compression_chunk_size.h"
 #include "ts_catalog/hypertable_compression.h"
 
+#define DECOMPRESSION_MARKER_START "::timescaledb-decompression-start"
+#define DECOMPRESSION_MARKER_END "::timescaledb-decompression-end"
+
+#if PG14_GE
+#define WriteLogicalReplicationMsgDecompressionStart()                                             \
```

Comment:
I don't really understand why the need to use a `#define` for this - can't you just declare an inline method?
that way you will
* have the `#if PG14_GE` inside the method
* no `\` madness

Reply:
No real reason. I just found it more convenient to have one precompiler statement than 5. Happy to change it though.